modules/Microsoft.DataFactory/factories/readme.md

```diff
@@ -52,7 +52,7 @@
 | `gitRepositoryName` | string | `''` |  | The repository name. |
 | `gitRepoType` | string | `'FactoryVSTSConfiguration'` |  | Repository type - can be 'FactoryVSTSConfiguration' or 'FactoryGitHubConfiguration'. Default is 'FactoryVSTSConfiguration'. |
 | `gitRootFolder` | string | `'/'` |  | The root folder path name. Default is '/'. |
-| `integrationRuntime` | _[integrationRuntime](integrationRuntime/readme.md)_ object | `{object}` |  | The object for the configuration of a Integration Runtime. |
+| `integrationRuntimes` | array | `[]` |  | An array of objects for the configuration of an Integration Runtime. |
```

Comment:
I know it sounds counterintuitive, but we should stick with the 'singular' name as the resource type uses singular too. If we don't, we see the above effect that the ReadMe generator doesn't link to the child folder anymore

Reply:
Since the child resource type is plural ([Microsoft.DataFactory/factories/integrationRuntimes](https://docs.microsoft.com/en-us/azure/templates/microsoft.datafactory/factories/integrationruntimes?pivots=deployment-language-bicep)) we should probably have also the child folder named as `integrationRuntimes`. If we named both the folder and the input parameter with the plural form, would that issue above with the readme be solved?

Reply:
I've renamed the folder to be plural as per the ARM reference docs and how our modules are named.